docs/imagecustomizer/api/configuration/input.md

```diff
@@ -4,18 +4,36 @@ parent: Configuration
 
 # input type
 
-Specifies the configuration for the input image.
+Specifies the configuration for the input image and associated files required for customizing.
 
 Example:
 
 ```yaml
 input:
   image:
     path: ./base.vhdx
+  verityHashes:
+    path: /input/verityhashes
+  ukis:
+    path: /input/ukis
```

Comment:
This is applicable to the other paths as well, if these are meant as directory paths, how do you decide to what include? E.g., how do you tell what file represents a verity hash, which one is a UKI and which one is something to ignore?